packages/tocco-util/src/nice/utils.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-undef */
```

Comment:
`eslint-disable` could be removed, if call on line 6 was `return window.app.getRunEnv()`?